src/js/plugins/plugin.autoplay.js

```diff
@@ -0,0 +1,130 @@
+/*global BookReader */
+
+/**
+ * Plugin which adds an autoplay feature. Useful for kiosk situations.
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enableAutoPlayPlugin: true
+});
+
+/**
+ * @augments BookReader.setup
+ */
+BookReader.prototype.setup = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    this.auto      = false;
+    this.autoTimer = null;
+    this.flipDelay = 5000;
+  };
+})(BookReader.prototype.setup);
+
+/**
+ * @augments BookReader.init
+ */
+BookReader.prototype.init = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    if (!this.options.enableAutoPlayPlugin) return;
+    this.bind(BookReader.eventNames.stop, (e, br) => br.autoStop());
```

Comment:
No longer need to pass `br` at all :) 

```suggestion
    this.bind(BookReader.eventNames.stop, () => this.autoStop());
```